<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="21" ignored="0" total="21" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="com.stepDefinition.CompaniesTest" duration-ms="1709" started-at="2022-03-22T10:50:25 IST" finished-at="2022-03-22T10:50:27 IST">
    <groups>
      <group name="cucumber">
        <method signature="FreeCRMRunner.Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" class="com.runner.FreeCRMRunner"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Command line test" duration-ms="1709" started-at="2022-03-22T10:50:25 IST" finished-at="2022-03-22T10:50:27 IST">
      <class name="com.stepDefinition.CompaniesTest">
        <test-method status="PASS" signature="setUpCucumber()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpCucumber" is-config="true" duration-ms="817" started-at="2022-03-22T10:50:25 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpCucumber -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="84" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether user is able to add new filters."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add filter to company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="11" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether user is able to apply filters on companies added to view them"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add filter to company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="1" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="13" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether user is able to remove filters if required."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add filter to company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="8" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether redirection happens to create companies form after clicking on create."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of create a company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="11" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate if the user has entered details in the all mandatory fields"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of create a company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="13" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether user is able to cancel the creation of companies is required"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of create a company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="8" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether user is able to unable to create company if  Mandatory details not filled."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of create a company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="6" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether valid mobile number is accpeting only digits in create company"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of create a company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="1" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="7" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether user is able to delete company after clicking on delete button/"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of delete a company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="1" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="8" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate user is able to cancel the deletion process if required."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of delete a company functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="8" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate that redirection happens to the Contact creation page if the create Contact button is clicked"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="7" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether the user has entered First Name and Last Name."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="5" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether the user is able to select Company from the already created Companies."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="5" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether phone number has 10 digits."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="6" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether the Contact detail page appears after giving the mandatory fields and clicking on Save"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="1" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="6" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether the user can Cancel the creation of Contact if required."]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="1" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="4" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate that the user is able to delete an existing Contact"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of delete Contact(s) functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="1" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="7" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate whether the user is able to Cancel the deletion process if requred"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of delete Contact(s) functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="6" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate that the details page of the selected Contact is opened when the user clicks on the edit tasks option"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of edit contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="1" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="3" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate if the edited details are saved for the contact which was edited by the user"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of edit contact(s) functionality.]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="FAIL" signature="Scenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="Scenario" duration-ms="5" started-at="2022-03-22T10:50:26 IST" description="Runs Cucumber Feature" data-provider="Scenarios" finished-at="2022-03-22T10:50:26 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA["To validate that the user can filter the contacts based on various criterias"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Verification of add filter to Contact functionality]"]]>
              </value>
            </param>
          </params>
          <exception class="io.cucumber.core.runner.DuplicateStepDefinitionException">
            <message>
              <![CDATA[Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.stepDefinition.AddContact.user_clicks_on_cancel() and com.stepDefinition.CompaniesTest.user_clicks_on_cancel()
at io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:277)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:271)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$3(CucumberExecutionContext.java:151)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:151)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at com.runner.FreeCRMRunner.Scenario(FreeCRMRunner.java:71)
at sun.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- io.cucumber.core.runner.DuplicateStepDefinitionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Scenario -->
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="setUpClass" is-config="true" duration-ms="0" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:com.stepDefinition.CompaniesTest@13c27452]" name="tearDownClass" is-config="true" duration-ms="574" started-at="2022-03-22T10:50:26 IST" finished-at="2022-03-22T10:50:27 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.stepDefinition.CompaniesTest -->
    </test> <!-- Command line test -->
  </suite> <!-- com.stepDefinition.CompaniesTest -->
</testng-results>
